src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
A question that should be addressed by reviews from the pros: should loggers be "global"s for util files?
I honestly don't know which is better.